tools/jsonproc/.gitignore

```diff
@@ -0,0 +1 @@
+jsonproc
```

Comment:
jsonproc should be another repository and should be included as a submodule instead. Alternatively, add it as required dependency since it seems to have public binary builds that can easily be installed, in which case this should be documented in INSTALL.md.

Reply:
This requested change is out of scope for this PR.